src/main/resources/templates/friends.html

```diff
@@ -0,0 +1,27 @@
+{# The page to display user's friends #}
+{% extends "base/li-sidebar-page" %}
+{% block title %} Exception Null - Buddy List {% endblock %}
+
+{% macro friendCard(imageSrc, username, userType, online, rating) %}
+<div class="card bottom-aligned">
```

Comment:
The `bottom-aligned` class aligns inline-block elements to the bottom of a fixed-height parent and needs to be used in conjunction with the `bottom-aligner` class. See [this](https://github.com/DumplingLTD/ExceptionNull/blob/master/src/main/resources/templates/base/li-nav-page.html#L13) example.

Instead, just add `display: inline-block; vertical-align: top;` to the `.friends .card` class.